_posts/0000-01-02-Kathyeb1773.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
```

Comment:
Welcome to CATaclysmically Cute! 
![CAT-aclysmically Cute logo](https://user-images.githubusercontent.com/85973518/122145964-cb965d00-ce1b-11eb-9db1-f385f56cc81e.png)